### ISSUE TITLE
Localization in editable not properly applied

### ISSUE THREAD

**Issue by Steinar30** [Shmew/Feliz.MaterialUI.MaterialTable#6]
## Description
Localization options for submodules of body not properly applied.

## Steps to reproduce
1. Based on the [editable row example](https://shmew.github.io/Feliz.MaterialUI.MaterialTable/#/MaterialTable/Examples/Editable)
2. Add the following import `open Feliz.MaterialUI.MaterialTable.localization.body`, to be able to compile
3. Add the following localization to the example: 

```
materialTable.localization [
            localization.body [
                localization.body.editTooltip "Custom edit tt"
                localization.body.deleteTooltip "Custom delete tt"
                localization.body.addTooltip "Custom add tt" 
                localization.body.editRow [
                    editRow.saveTooltip "Custom save edit tt"
                    editRow.deleteText "Custom delete text"
                ]
            ]
]
```
4. Notice that the localization applies to the edit, add and delete buttons.
5. Notice that when pressing delete, the localization for deleteText does not apply.
6. Notice that when pressing edit, the localization for saveToolTip does not apply.

## Expected behavior
Tooltip for save in edit should be "custom save edit tt" instead of Save.
Confirmation message for delete should be "Custom delete text"

## Screenshots
![image](https://user-images.githubusercontent.com/31291113/116045914-d2ea8700-a672-11eb-9bfb-323a86b26a55.png)

## Additional context
The cause of this could be that the localization module has a static type body and module body. No expert in this but the compiler could possibly confuse them. 

I am aware that opening the body module like this is probably not the intended behavior. Removing it the compiler shows the following: 
![image](https://user-images.githubusercontent.com/31291113/116046503-7045bb00-a673-11eb-9cb1-57181b2f8dcb.png)

Notably this seems to only happen with the submodules of the body module, editRow and filterRow.

**Comment by Shmew** [comment]
Ah yeah, that's an issue with how the F# compiler handles very nested modules with required namespacing. I'll get a fix out for this shortly.

**Comment by Shmew** [comment]
I just released version 1.9.0 that resolves this issue. In your example the new API would be like this:

```fsharp
materialTable.localization [
    localization.body [
        body.editTooltip "Custom edit tt"
        ...
        body.editRow [
            editRow.saveTooltip "Custom save edit tt"
            ...
        ]
    ]
]
```
